tests/lib/external/snapd-testing-tools/tools/snaps.name

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: snaps.name gadget, kernel, core"
+    echo ""
+    echo "get the name of a specific snap for the current system"
+}
+
+gadget_name() {
+    echo "$(snap list | grep 'gadget$' | awk '{ print $1 }')"
+}
+
+kernel_name(){
+    echo "$(snap list | grep 'kernel$' | awk '{ print $1 }')"
```

Comment:
```suggestion
    # TODO this may not always be true during remodel scenarios
    # if the old kernel remains
    echo "$(snap list | grep 'kernel$' | awk '{ print $1 }')"
```